lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -345,8 +345,10 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
           </div>
         %endif
 
-        % if course_program_info and course_program_info['category']=='xseries':
-            <%include file = "_dashboard_xseries_info.html" args="course_program_info=course_program_info, enrollment_mode=enrollment.mode" />
+        % if course_program_info and course_program_info.get('category')=='xseries':
+          %for program_data in course_program_info.get('course_program_list', {}):
```

Comment:
i guess it won't make a difference in display, but `course_program_list` would be a list (`[]`) instead of a dict right?